src/docfx/config/ops/OpsMetadataRuleConverter.cs

```diff
@@ -161,45 +162,71 @@ private static bool TryGetAttributeCustomRules(Dictionary<string, OpsMetadataRul
             return attributeCustomRules.Count != 0;
         }
 
-        private static bool TryGetAllowlist(
-            string parentName, string? listId, Allowlists allowlists, int index, out Dictionary<string, EnumDependenciesSchema?> allowList)
+        private static bool TryGetAllowlist(string attribute, string? listId, AllowLists allowlists, out Dictionary<string, EnumDependenciesSchema?> allowList)
         {
             allowList = new Dictionary<string, EnumDependenciesSchema?>();
 
-            if (!string.IsNullOrEmpty(listId) && allowlists.TryGetValue(listId, out var mainAllowlist))
+            if (!string.IsNullOrEmpty(listId) && allowlists.TryGetValue(listId.Substring("list:".Length), out var subAllowlist))
             {
-                if (string.IsNullOrEmpty(mainAllowlist.DependentAttribute))
+                if (string.IsNullOrEmpty(subAllowlist.NestedValue))
                 {
-                    allowList = mainAllowlist.Values.Keys.ToDictionary(validValue => validValue, validValue => (EnumDependenciesSchema?)null);
+                    allowList = subAllowlist.NestedTaxonomy.list.ToDictionary(x => x, x => (EnumDependenciesSchema?)null);
```

Comment:
`allowList` is being repeatedly initialized, consider restructure code to avoid this?

Reply:
so careful.

Reply:
allowlist is out, are you sure?